js/ui/drop_down_editor/ui.drop_down_button.js

```diff
@@ -10,7 +10,13 @@ const DROP_DOWN_EDITOR_BUTTON_VISIBLE = 'dx-dropdowneditor-button-visible';
 
 const BUTTON_MESSAGE = 'dxDropDownEditor-selectLabel';
 
-export default class ClearButton extends TextEditorButton {
+export default class DropDownButton extends TextEditorButton {
+    constructor(name, editor, options) {
+        super(name, editor, options);
+
+        this.currentTemplate = null;
```

Comment:
this._currentTemplate ?

Reply:
Nah, It's a private Class. Inner fields of [TextEditorButton](https://github.com/DevExpress/DevExtreme/blob/9d45edd6c9acde880470d8cadf09ad4cf4c641e9/js/ui/text_box/texteditor_button_collection/button.js#L4) does not have a such prefix too. I'd prefer to use [native support](https://github.com/tc39/proposal-class-fields#private-fields) of Class' private fields in the future